### DISCUSSION TITLE
"Load more ..." in issues fails to load more items

### DISCUSSION THREAD

**Discussion by SebastianZimmeck** [community/community#23183] [Projects and Issues]
<p>For an issue with lots of items that cannot be all displayed, the “Load more …” link does not work. <a href="https://github.com/privacycg/proposals/issues/10" rel="noopener nofollow ugc">Here</a> is the issue where I am experiencing that behavior. I tried with Chrome, Brave, and Firefox, but it did not work for either. This does not seem to be related to any browser extensions, ad blockers, etc.</p>
<p>Is anyone else experiencing this behavior? Any suggestions?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/8/383053098927632910739dfe50f09190652de7b4.png" data-download-href="/uploads/short-url/814gUIboZgHghnQzNLTQ1hmyCDW.png?dl=1" title="Screen Shot 2021-01-28 at 10.50.19 AM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/11034003/181075481-624c7fdb-815c-44ae-8aea-7b33e83daaa4.png" alt="Screen Shot 2021-01-28 at 10.50.19 AM" data-base62-sha1="814gUIboZgHghnQzNLTQ1hmyCDW" width="690" height="102" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/8/383053098927632910739dfe50f09190652de7b4_2_690x102.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/8/383053098927632910739dfe50f09190652de7b4_2_1035x153.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/3/8/383053098927632910739dfe50f09190652de7b4_2_1380x204.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/8/383053098927632910739dfe50f09190652de7b4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2021-01-28 at 10.50.19 AM</span><span class="informations">1854×276 8.86 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by rburkey2005** [comment, marked as answer]
<p>Yes, I have experienced it, today.  <a href="https://github.community/t/load-more-and-issues-with-a-lot-of-comments/158141">I had just put up a message about it myself</a>.</p>
<p>It started working for me right after I had posted a new comment in my issue.  Maybe a coincidence.  Maybe posting a new comment unsticks “Load more” somehow.  Who knows?</p>

**Comment by SebastianZimmeck** [comment]
<p>Thanks, <a class="mention" href="/rburkey2005">@rburkey2005</a>, good to know. I just tried it on the issue where I had this problem, and it works now. For me, I did not need to post a new comment. But as you say, who knows?</p>
